strider/fetcher.py

```diff
@@ -118,7 +129,7 @@ async def generate_from_kp(
             onehop_response = await kp.solve_onehop(
                 onehop_qgraph,
             )
-            await save_kp_onehop(kp.id, onehop_qgraph, onehop_response)
+            await save_kp_onehop(kp.id, onehop_qgraph, onehop_response.dict())
```

Comment:
why dict() here? should we defer that until later?

Reply:
This is where we save the response to the cache. I'm not aware that a reasoner-pydantic object can be gzipped and then saved into redis.

Reply:
🫡